.github/workflows/deploy-and-push-image.yml

```
@@ -1,4 +1,4 @@
-name: Create and publish Docker images
+name: Deploy Application and publish Docker Images
```

```suggestion
name: Publish Docker Images and Deploy Application
```

nit: since we need the docker images before deployment